File: examples/Fortran/hello.f90
! (c) 2011 University of Houston System.  All rights reserved.


!
! the ubiquitous hello world program
!

program whoami

  include 'shmem.fh'

  integer npes, me
  character*32 h

  call start_pes(0)

  npes = num_pes()
  me = my_pe()
  call hostnm(h)

  print *, h, 'I am ', me, ' of ', npes

end program whoami

File: examples/C/hello.c
/* (c) 2011 University of Houston.  All rights reserved. */


#include <stdio.h>

#include <mpp/shmem.h>

int
main(int argc, char **argv)
{
  int me, npes;

  start_pes(0);

  me   = _my_pe();
  npes = _num_pes();

  printf("Hello from node %4d of %4d\n", me, npes); 

  return 0;
}
